tools/modules/framework/interface_categories.sh

```
@@ -0,0 +1,165 @@
+
+	local -A description=(
+		["Kernel"]="Alternative kernels, headers, rolling updates, overlays"
+		["Storage"]="Install to internal media, ZFS, NFS, read-only rootfs"
+		["Access"]="Manage SSH daemon options, enable 2FA"
+		["User"]="Change shell, adjust MOTD"
+		["Updates"]="OS updates and distribution upgrades"
+	)
+
+	for key in "${keys[@]}"; do
+		eval "desc=\"$key - ${description[$key]}\""
+		echo -e "$desc"
```

Shellcheck (0.10.0)

[warning] desc is referenced but not assigned.

(SC2154)